packages/app-mobile/components/NoteEditor/commandDeclarations.ts

```diff
@@ -34,6 +48,63 @@ const declarations: CommandDeclaration[] = [
 	{
 		name: 'editor.execCommand',
 	},
+
+	{
+		name: EditorCommandType.ToggleBolded,
+		label: () => _('Bold'),
+		iconName: 'material format-bold',
+	},
+	{
+		name: EditorCommandType.ToggleItalicized,
+		label: () => _('Italic'),
+		iconName: 'material format-italic',
+	},
+	...headerDeclarations(),
+	{
+		name: EditorCommandType.ToggleCode,
+		label: () => _('Code'),
+		iconName: 'material code-json',
+	},
+	{
+		name: EditorCommandType.ToggleMath,
```

Comment:
Currently, `textMath` and the header commands aren't recognized as editor commands by `isEditorCommand`. It may make sense to prefix these commands with `editor.` here (e.g. <code>`editor.${EditorCommandType.ToggleMath}`</code>) so that they are recognized as editor commands.